mmv1/templates/terraform/examples/apphub_application_full.tf.erb

```diff
@@ -0,0 +1,38 @@
+data "google_project" "host_project" {}
+
+resource "google_project_service" "apphub" {
```

Comment:
We can enable a project_service in the CI project itself to prevent needing an extra project per test. Please remove this

Reply:
As per my understanding, it needs to be added to https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/infra/terraform/main.tf#L161
Fixed accordingly.

Reply:
Great!